app/assets/javascripts/discourse/app/templates/user.hbs

```diff
@@ -384,7 +384,7 @@
                       @model={{this.model.invited_by}}
                     >{{this.model.invited_by.username}}</LinkTo></dd></div>
               {{/if}}
-              {{#if this.model.trust_level}}
+              {{#if this.hasTrustLevel}}
```

Comment:
```suggestion
              {{#if (gte this.model.trust_level 0)}}
```

Reply:
Well answering my own question... not it doesnt...

```
{{#if (gte null 0)}}
  IT WILL SHOW
{{/if}}
```

:/

Reply:
it is at this point that my brain always replays https://www.destroyallsoftware.com/talks/wat 🤣

Reply:
There's a special place in my heart for people liking and linking to gary bernhardt ❤️ 

I really recommend this talk if you have never seen it: https://www.destroyallsoftware.com/talks/ideology